Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.9.0-alpine
+MAINTAINER Lukasz Gornicki <derberg@wp.pl>
+# add bash and git
+RUN apk update && apk add --no-cache bash git openssh nano && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm /var/cache/apk/*
+# set what needs to be included in the image
+COPY . /docpad-skeleton-apidocs/
```

Comment:
While dockerize node project, good practice is firstly copy only `package.json` and run `npm install`, only after copy the whole application. Thanks to this if you change application, docker will get dependencies (`node_modules`) from cache, so build phase is MUUUUCH faster.

```docker
# install dependencies
COPY package.json /app/
RUN npm install --silent

# copy application
COPY . /app/
```

but in this case we don't have only npm dependencies, so you could try:
```docker
# install dependencies
COPY package.json bower.json .bowerrc  /app/
RUN npm run install-deps

# prepare dependencies
RUN npm run prepare-dependencies

# copy application
COPY . /app/
```
where `prepare-dependencies` is npm script: `npm run copy:modernizr && npm run copy:zeroclipboard && npm run build:lunrplugin && gulp getDependencyInteractiveDocu && del-cli src/raw/bower_components/list.js/docs`

Developer experience with this change would be much better, but at this moment it's not needed, so this is more a proposal for the future.